Modules/Microsoft365DSC/DSCResources/MSFT_IntuneMobileAppConfigurationPolicyIOS/settings.json

```diff
@@ -0,0 +1,32 @@
+{
+    "resourceName": "IntuneMobileAppConfigurationPolicyIOS",
+    "description": "This resource configures an Intune Managed Device Mobile App Configuration Policy for iOS Device.",
+    "permissions": {
+        "graph": {
+            "delegated": {
+                "read": [
+                    {
+                        "name": "DeviceManagementApps.Read.All"
+                    }
+                ],
+                "update": [
+                    {
+                        "name": "DeviceManagementApps.ReadWrite.All"
+                    }
+                ]
+            },
+            "application": {
+                "read": [
+                    {
+                        "name": "DeviceManagementApps.Read.All"
+                    }
+                ],
+                "update": [
+                    {
+                        "name": "DeviceManagementApps.ReadWrite.All"
+                    }
+                ]
+            }
+        }
```

Comment:
Because we're dealing with Intune assignments, an additional entry for `Group.Read.All` is required. You can take at the other Intune resources.

Reply:
should be resolved in latest pull request